datahub-project/datahub
# Column level lineage and path test

Test steps:
1. Open Dataset with column-level lineage configured
2. Navigate to Lineage tab -> Visualize Lineage
3. Enable “Show Columns” switch
4. Verify the functionality of column lineage
5. Open dataset Impact Analysis view, enable Column Lineage
6. Verify Upstream and Downstream column lineage
7. Verify Column Path modal works as expected

## Checklist
